recipes/refinitiv-dataplatform/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "refinitiv-dataplatform" %}
+{% set version = "1.0.0a4" %}
```

Comment:
We don't usually allow alpha versions.  Is there a full release that is available?

Reply:
@scopatz The full release is not available yet. I am not sure how long before the full release will be available.
Could we publish on difference channel not a main channel? i.e. prerelease channel. What do you think?